### DISCUSSION TITLE
How to auto save when exit insert mode to normal mode?

### DISCUSSION THREAD

**Discussion by Dennis-Zhang-SH** [ayamir/nvimdots#976] [Q&A] [answered]
as title

**Comment by charliie-dev** [comment, marked as answer]
https://github.com/pocco81/auto-save.nvim

it is truly a sad thing to see someone dont have access to search engines.

**Comment by Dennis-Zhang-SH** [comment]
> https://github.com/pocco81/auto-save.nvim
> 
> it is truly a sad thing to see someone dont have access to search engines.

Ha ha my bad, it's just the comunity over here is so welcome, that I like to put up the questions here.
